frontends/qt/server/Makefile

```diff
@@ -1,5 +1,17 @@
+SECURITYFLAGS = " -O2 -D_FORTIFY_SOURCE=2 "
+SECURITYFLAGS += " -fstack-protector-all -fPIC -fPIE"
+SECURITYFLAGS += " -fstack-check "
+EXTLDFLAGS:="-Wl,-z,now,-z,relro,-z,noexecstack,-z,noexecheap,-z,defs,-pie,-s,-E"
+CXXFLAGS += ${SECURITYFLAGS}
+CFLAGS += ${SECURITYFLAGS}
+LFLAGS += ${EXTLDFLAGS}
+export CXXFLAGS
+export CFLAGS
+export LFLAGS
+export CGO_LDFLAGS_ALLOW=".*"
+export CGO_CFLAGS_ALLOW=".*"
 build:
-	CGO_ENABLED=1 go build -ldflags="-s -w" -buildmode=c-shared -o libserver.so
+	CGO_ENABLED=1 go build -buildmode=c-shared -ldflags -extldflags=${EXTLDFLAGS} -o libserver.so
```

Comment:
This does not work on macOS unfortunately: `running clang failed: exit status 1: unknown option -z`.

Please fix by making separate targets for now. We can fix if for macOS later.

Reply:
Could you share the version of OS X, clang, ld, and golang? If you edit the EXTLDFLAGS, what works?